src/extra/themes/default/lv_theme_default.c

```diff
@@ -24,10 +24,10 @@
 #define LIGHT_COLOR_CARD       lv_color_white()
 #define LIGHT_COLOR_TEXT       lv_palette_darken(LV_PALETTE_GREY, 4)
 #define LIGHT_COLOR_GREY       lv_palette_lighten(LV_PALETTE_GREY, 2)
-#define DARK_COLOR_SCR         lv_palette_darken(LV_PALETTE_GREY, 4)
-#define DARK_COLOR_CARD        lv_palette_darken(LV_PALETTE_GREY, 3)
+#define DARK_COLOR_SCR         lv_color_hex(0x22252a)
```

Comment:
Can't we use palette functions here?

Reply:
I tried that at first, but the palette doesn't give nearly as much flexibility as it provides a very limited number of color shades. Since it is expected that the dark theme has a greyish-black background; I think it's safe enough to use hardcoded colors here. It can always be overriden with another style.

Alternatively, we could add an `LV_PALETTE_DARK_GREY` as a darker extension of `LV_PALETTE_GREY`.

Reply:
> I tried that at first, but the palette doesn't give nearly as much flexibility as it provides a very limited number of color shades. Since it is expected that the dark theme has a greyish-black background; I think it's safe enough to use hardcoded colors here. It can always be overriden with another style.

Okay, let's use the hard-coded values. :)